persistence/src/main/scala/za/co/absa/spline/persistence/tx/ArangoTx.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.persistence.tx
+
+import com.arangodb.ArangoDatabaseAsync
+
+import scala.concurrent.Future
+
+trait ArangoTx {
+  def execute(db: ArangoDatabaseAsync): Future[Unit]
```

Comment:
What is the aim of having this function ? Can't we use the executeAndReturn only ? If we don't want the result we just don't use it

Reply:
ok ok. Another question came in mind : If there is nothing to return shouldn't it be sync instead?

Reply:
Ok, as discussed I removed `executeAndReturn` along with `NativeQuery` class according to YAGNI.